docs: remove old readme content and reference new pipeline repo (#360)
WilhelmusLab/IceFloeTracker.jl: README.md

(IceFloeTracker) pkg> test
```

## Notebooks

> There are Jupyter notebooks in `IceFloeTracker.jl` that can be used as examples to access some of the image processing and tracking functions, available at `IceFloeTracker.jl/notebooks`. 

Reviewer: ```suggestion
There are Jupyter notebooks illustrating the main image processing and tracking functions in the `/notebooks` folder. 
```

(IceFloeTracker) pkg> test
```

> ## Notebooks

Author: We do have some Jupyter notebooks though, I added a statement just alerting users that they are available, but it would be up to the user to figure out how run a `.ipynb`.

You now have access to `IceFloeTracker` from inside your Pluto notebook!

> ## Using in a Pipeline

Reviewer: ```suggestion
## Interface for Pipeline Workflows
```

> See the related tools at `https://github.com/WilhelmusLab/ice-floe-tracker-pipeline.git` for a Julia Command-line Interface to access all the source functions and templates for using the Cylc pipeline orchestrator.

Reviewer: ```suggestion
See related tools in the [IFTPipeline repo](https://github.com/WilhelmusLab/ice-floe-tracker-pipeline#ice-floe-tracker-pipeline) including a Julia Command-line Interface and templates that leverage the [Cylc](https://cylc.github.io/) pipeline orchestrator.
```
